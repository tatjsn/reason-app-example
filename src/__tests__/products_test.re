open Jest;
open ExpectJs;

let _ =

describe "src/products.re" (fun _ => {

  test "is defined" (fun _ => {
    let component = ReactShallowRenderer.renderWithRenderer <Products products=["Prod1"] />;
    expect (Js.Undefined.return component) |> toBeDefined;
  });

  test "renders" (fun _ => {
    let tree = Products.createElement products::["Prod1"] children::[] ()
      |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });

});
