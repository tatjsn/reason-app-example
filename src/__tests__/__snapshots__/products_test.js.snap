// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/products.re renders 1`] = `
<div
  className="Products"
>
  Our products:
  <div>
    Prod1
  </div>
</div>
`;
